src/internal/connector/graph/service.go

```diff
@@ -61,6 +78,90 @@ func (s Service) Serialize(object absser.Parsable) ([]byte, error) {
 	return writer.GetSerializedContent()
 }
 
+// ---------------------------------------------------------------------------
+// Helpers
+// ---------------------------------------------------------------------------
+
+type clientConfig struct {
+	noTimeout bool
+}
+
+type option func(*clientConfig)
+
+// populate constructs a clientConfig according to the provided options.
+func (c *clientConfig) populate(opts ...option) *clientConfig {
```

Comment:
Request a name change. We have a lot of populate functions. It would be nice to distinguish from other methods.

Reply:
The populate funcs that I see are all methods on discrete structs, which gives distinct space for each one to clearly describe its behavior without conflict of other methods.  Changing the name here is akin to asking to change the pattern in all cases throughout the code.  Is that a change you believe will resolve confusion in long term maintenance of the app?